### PR TITLE
Adding silence and unsilence actions for sensu

### PR DESCRIPTION
**Pull request by DoriftoShoes**
This PR adds a silence and unsilence action to the Sensu pack.  This allows users to silence either an entire client or a specific check on a client.

*KNOWN ISSUES*
* expire is not being recognized

### CODE REVIEW

File: packs/sensu/actions/silence.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: silence
+runner_type: run-local-script
+description: Silence a Sensu client or check
+enabled: true
+entry_point: silence.py
+parameters:
+  check:
+    type: string
+    description: The check to silence
```
Comment (dzimine) [inline packs/sensu/actions/silence.yaml]: which are required? I assume EITHER client or check required, so suggest to put it in the description of both

File: packs/sensu/actions/silence.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  client:
+    type: string
+    description: The client to silence
+    required: true
+  expiration:
+    type: integer
+    description: Expiration date for silence stash
```
Comment (dzimine) [inline packs/sensu/actions/silence.yaml]: What format? Add example in description.

File: packs/sensu/actions/silence.yaml (hunk 1 above)
Comment (DoriftoShoes) [inline packs/sensu/actions/silence.yaml]: client is required, check is optional.  That is why I made client a required parameter ;)

**Comment by dzimine** [comment]
:cool:

**Pull request merged**